orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/config/CloudDriverConfiguration.groovy

```diff
@@ -106,4 +109,10 @@ class CloudDriverConfiguration {
       .build()
       .create(KatoRestService)
   }
+
+  @Bean
+  ObjectMapper pipelineObjectMapper() {
+    new ObjectMapper()
+      .registerModule(new SimpleModule("MyConverterModule").addAbstractTypeMapping(Stage, PipelineStage))
```

Comment:
You might have been able to get away with just doing a

`private static final ObjectMapper objectMapper = new ObjectMapper()....` in `DeploymentDetailsAware` and not creating a new bean and passing it all around.